Sprint-1/implement/dedupe.test.js

```diff
@@ -13,15 +13,15 @@ E.g. dedupe([1, 2, 1]) target output: [1, 2]
 
 // Acceptance Criteria:
 
-// Given an empty array
-// When passed to the dedupe function
-// Then it should return an empty array
-test.todo("given an empty array, it returns an empty array");
-
-// Given an array with no duplicates
-// When passed to the dedupe function
-// Then it should return a copy of the original array
-
-// Given an array with strings or numbers
-// When passed to the dedupe function
-// Then it should remove the duplicate values, preserving the first occurence of each element
+test("Given an empty array then it should return an empty array", () => {
+  expect(dedupe([])).toBe([]);
+});
+
+
+test("Given an array with no duplicates then it should return a copy of the original array", () => {
+  expect(dedupe([1,2,3])).toBe([1,2,3]);
```

Comment:
We must use `.toEqual()` instead of `.toBe()` when comparing arrays or objects; `.toBe()` does not compare two arrays by their values. Since `toEqual()` works equally well on primitive values such as number, and string, you can also just use `.toEqual()` to compare all types of values.

In JS, an array is a kind of ***object***, and objects are compared by ***references*** instead of by values.

To learn more about how arrays/objects are compared/copied in JS, and to learn more about how to compare two arrays by their values, may I suggest asking ChatGPT
- Why in JS `[1, 2, 3] === [1, 2, 3]` is `false`?" 
- How to compare two arrays by value in JS?